frontend/src/app/modules/fields/edit/field-types/work-package-edit-field.component.ts

```diff
@@ -27,16 +27,52 @@
 // ++
 
 import {Component} from "@angular/core";
-import {EditFieldComponent} from "core-app/modules/fields/edit/edit-field.component";
+import {SelectEditFieldComponent, ValueOption} from './select-edit-field.component';
+import {ApiV3FilterBuilder} from "core-components/api/api-v3/api-v3-filter-builder";
+import {WorkPackageResource} from "core-app/modules/hal/resources/work-package-resource";
+import {DebouncedRequestSwitchmap, errorNotificationHandler} from "core-app/helpers/rxjs/debounced-input-switchmap";
+import { take, skip } from 'rxjs/operators';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/c707e7bf78dbad7cac984139a18fb9d109e530df/frontend/src/app/modules/fields/edit/field-types/work-package-edit-field.component.ts#L35"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>